### PR TITLE
commit changes

### PR DESCRIPTION
**Pull request by andrewmpho47**
Waiting for Trevor to approve the pull request

### CODE REVIEW

**Comment by TrevorMal** [comment]
Pull request you tell the reviewer what changes are being introduced.

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (TrevorMal) [inline .idea/vcs.xml:1]: IDE Configurations files should be excluded and you make use of .gitignore file to excluded certain files.

**Comment by TrevorMal** [comment]
Am seeing also log files those don't belong in the repo.

**Review by TrevorMal — CHANGES_REQUESTED** [review]

**Comment by TrevorMal** [comment]
Check code styling and access modifiers for some beans.

File: .idea/vcs.xml (hunk 1 above)
Comment (andrewmpho47) [reply .idea/vcs.xml:1]: Managed to use gitignore to untrack the IDE configurations and deleted logs that where auto generated by IntelliJ

**Pull request merged**